config/environments/test.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.configure do
+  ActiveSupport::Deprecation.silenced = true
```

Comment:
was this a noise thing? I'd kinda like to know about these but not if they are super noisy.